modules/ROOT/pages/design-create-publish-api-specs.adoc

```diff
@@ -5,11 +5,15 @@ endif::[]
 :imagesdir: ../assets/images
 :page-aliases: general:getting-started:design-an-api.adoc, design-migrating-ms.adoc
 
-API Designer provides a web-based interface for designing, documenting, testing, and sharing API specifications and fragments.
+API Designer provides a web-based interface for designing API specifications and fragments in:
```

Comment:
Be sure that your intro text tells what this is and why "you" want to use it, to accomplish what goals.
Active voice, direct address.

Reply:
Thanks, Kim. As we discussed in the Slack chat, this is a legacy topic that I simply updated to add the AsyncAPI support. I had to make minor edits to make the text work, but otherwise did not edit the topic. I will open a Jira issue to address the organization and style suggestions so that we can address those separately from the AsyncAPI support additions.